### PR TITLE
Address first six issues from issue #35

### PR DESCRIPTION
**Pull request by RishabhC-137**
This pull request resolves the first six issues as mentioned in [Issue #35](https://github.com/DiceDB/playground-web/issues/35):

- Updated README to add details for node version requirement and other prerequisites.
- Added steps in README to run prettier scripts in case of GitHub workflow failure.
- Renamed Dockerfile_Backend to PlaygroundMonoDockerfile and updated YAML files accordingly.
- Renamed CLI package to Shell.
- Removed comment: // comment this to hide footer from page.tsx file.
- Updated all comments to follow consistent formatting: // comment instead of //comment.

### CODE REVIEW

File: src/components/Playground/Playground.tsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ export default function Playground() {
               </div>
             </div>
             <div className="h-64 md:h-96 bg-gray-100 rounded-lg overflow-hidden shadow-md">
-              <Cli decreaseCommandsLeft={decreaseCommandsLeft} />
+              <Shell decreaseCommandsLeft={decreaseCommandsLeft} />{' '}
+              {/* Updated from Cli to Shell */}
```
Comment (lucifercr07) [inline src/components/Playground/Playground.tsx:47]: Do we require this comment in code?

File: docker-compose.yml [hunk 2]
```diff
@@ -9,20 +8,20 @@ services:
   backend:
     build:
       context: .
-      dockerfile: Dockerfile_Backend
+      dockerfile: PlaygroundMonoDockerfile  # Updated Dockerfile name
```
Comment (lucifercr07) [inline docker-compose.yml:11]: Do we require this comment in code?

**Review by lucifercr07 — CHANGES_REQUESTED** [review]
We don't need comments in code with // Updated *
Please resolve it

File: src/styles/globals.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
-@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@400;700&display=swap');
+@import url('https:// fonts.googleapis.com/css2?family=Assistant:wght@400;700&display=swap');
```
Comment (lucifercr07) [inline src/styles/globals.css:4]: Is this intentional?

**Comment by RishabhC-137** [comment]
@lucifercr07

**Comment by lucifercr07** [comment]
@RishabhC-137 conflicts are still there, please resolve once.

**Comment by RishabhC-137** [comment]
Working on it

**Pull request merged**